ngrinder-groovy/build.gradle

```diff
@@ -11,8 +11,18 @@ dependencies {
     compile (group: "org.codehaus.groovy", name: "groovy-all", version:"2.2.1")
     compile (group: "junit", name: "junit-dep", version:"4.8.2")
     compile (group: "org.hamcrest", name: "hamcrest-all", version:"1.1")
+    compile (group: "commons-io", name: "commons-io", version:"2.2")
+    compile (group: "commons-lang", name: "commons-lang", version:"2.6")
+    compile (group: "net.sf.grinder", name: "grinder", version:"3.9.1") {
+        exclude (module: "clojure")
+        exclude (module: "jython")
+        exclude (module: "grinder-xmlbeans")
+        exclude (module: "jedit-syntax")
+        exclude (module: "asm")
+    }
 
     testCompile (group: "org.mockito", name: "mockito-core", version:"1.8.5")
+    testCompile (group: "asm", name: "asm", version:"3.3.1")
```

Comment:
Why should this be testCompile scope?

Reply:
Because groovy module needs this lib when test.

If pass the test using the lib excluded above, I'll use it.